contracts/tests/sbtc-deposit.test.ts

```diff
@@ -321,7 +321,7 @@ describe("optimization tests", () => {
     const { burnHeight, burnHash } = getCurrentBurnInfo();
 
     const totalAmount = 1000000n;
-    const runs = 650;
+    const runs = 400;
```

Comment:
Hmm, not entirely sure where that diff comes from? Should we change the list parameter size to 400 as well in the contracts?

Reply:
We updated the contract calls to take in more data, write more, and emit more data. I would imagine the diff comes from that. I'm not sure if this will be a production issue as well, where 650 is unusable. If so, we should lower it.

Reply:
Where does the 650 come from?

Reply:
@hstove I remember you working on these optimization PRs, any idea here?

Reply:
My guess is that it is close to maximum allowed before the report returned an error.